package.json

```diff
@@ -70,7 +71,7 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "serve": "http-serve ./build -o",
+    "serve": "serve build --single --listen 8080",
```

Comment:
Do we actually need the `serve` package and to run it here in this way instead of `node ./serve.js`? Asking because we encourage the users to install the tool globally with `npm i @ethersphere/bee-dashboard -g` and to run it as `bee-dashboard serve`. That means they actually run the `serve` command in the `bin` part of this file and not this one defined in `scripts`. Could we remove the `serve` package and unify? (I am asking as I genuinely don't know)

https://github.com/ethersphere/bee-dashboard/pull/122/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R16

Reply:
I dont think we need the serve package but we would still need the core dependency of `serve-handler` to run the serve.js file. We can pull it out if you would like as like you said we dont really need it in the current capacity that we run the dashboard.

Reply:
I would actually prefer that. So let's add the `serve-handler`, remove the `serve` dependency and change this line to `"serve": "node ./serve.js"`

BTW it is always recommended to add any dependencies one uses for two reasons:
- at some point we may decide to remove the package that includes it as a dependency and therefore break it
- the package that includes it as a dependency gets updated and removes it

This reminds me we should add `depcheck` that will catch these things.